sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -757,12 +757,13 @@ export class ClientContext {
       options: requestContext.options,
       partitionKeyRangeId: requestContext.partitionKeyRangeId,
       useMultipleWriteLocations: this.connectionPolicy.useMultipleWriteLocations,
-      partitionKey: requestContext.partitionKey,
+      partitionKey: requestContext.partitionKey !== undefined ? mapPartitionToInternal(requestContext.partitionKey) : undefined,
```

Comment:
can we rename this method mapPartitionToInternal ?
Maybe something like convertToInternalPartitionKey() or something more intuitive?

Reply:
Ack.